src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```
`health` is not configurable, 100 by default for all animals